inc/Integrations/Google/Sheets/GoogleSheetsIntegration.php

```diff
@@ -18,6 +18,11 @@ public static function register_blocks(): void {
 
 		foreach ( $data_source_configs as $config ) {
 			$data_source = GoogleSheetsDataSource::from_array( $config );
+	
+			if ( false === $config['service_config']['enable_blocks'] ) {
```

Comment:
add coalescing here too (with parens)

Reply:
repeat for other instances

Reply:
This should be fine. An explicit `false` value is the only time we want to skip out. If it's `undefined` we want to default to enabling blocks.

Reply:
Yes, but will generate a warning without coalescing or explicit check.